napari/layers/labels/_tests/test_labels_mouse_bindings.py

```diff
@@ -63,14 +63,13 @@ def test_paint(Event, brush_shape, expected_sum):
 
 
 @pytest.mark.parametrize(
-    "brush_shape, expected_sum", [("circle", 244), ("square", 274)]
+    "brush_shape, expected_sum", [("circle", 240), ("square", 250)]
 )
 def test_paint_scale(Event, brush_shape, expected_sum):
     """Test painting labels with circle/square brush when scaled."""
     data = np.ones((20, 20))
     layer = Labels(data, scale=(2, 2))
-    layer.brush_size = 10
-    assert layer.cursor_size == 20
+    layer.brush_size = 20
```

Comment:
@sofroniewn can you explain why this test changed?

Reply:
brush size is now in world coordinates, not in data coordinates, and cursor_size is now always identical to brush size. We don't actually need both anymore - one can be dropped from the layer. Probably cursor_size which is on the base and basically never used anymore